File: Ohtap E Commerce Domain Models/Development/PHP/Objects/Currencies/CurrencyBasic.php
<?php
namespace Ohtap\ECommerce\DomainModels\Objects\Currencies;





class CurrencyBasic implements ICurrencyData
{
    public string $ID;
    public string $Symbol;
    public  string $FundCode;
    public string $Name;
    public int $State;
    public int $OrderNo;
    public string $Description;
}

File: Ohtap E Commerce Domain Models/Development/NetFramework/Models Objects/Models Objects/Currencies/CurrencyBasic.cs
using Ohtap.ECommerce.Domain.Models.Objects.Units;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Ohtap.ECommerce.Domain.Models.Objects.Currencies
{
    public class CurrencyBasic : ICurrencyData
    {
        public virtual string ID { get; set; }
        public virtual string Code { get; set; }
        public virtual string Name { get; set; }
        public virtual int Order { get; set; }
        public virtual string Description { get; set; }
        public virtual int State { get; set; }
        public virtual Units.UnitBasic Unit { get; set; }
        public virtual string Symbol { get; set; }
        public virtual string FundCode { get; set; }

        IUnitData ICurrenciesUnit.Unit { get { return this.Unit; } set { this.Unit = (Units.Unit)value; } }
    }
}
